src/index.html

```diff
@@ -11,6 +11,37 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <!-- <h1>Search bar airbnb</h1> -->
+    <div class="search-bar-big">
+      <div class="search-container-big">
```

Comment:
why do you need so many divs? you can set all styles to the form directly

Reply:
fixed that, cleaned up one div in every search bar, thank you